FEXCore/Source/Interface/Core/JIT/JIT.cpp

```diff
@@ -711,6 +712,46 @@ CPUBackend::CompiledCode Arm64JITCore::CompileCode(uint64_t Entry, const FEXCore
   adr(TMP1, &JITCodeHeaderLabel);
   str(TMP1, STATE, offsetof(FEXCore::Core::CPUState, InlineJITBlockHeader));
 
+  if (CheckTF) {
+    ARMEmitter::SingleUseForwardLabel l_TFUnset;
```

Comment:
Could the implementation details of this be put in some function like `EmitTFCheck` or something?